// Copyright 2019 The Chromium Authors
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

syntax = "proto3";

option java_multiple_files = true;
option java_package = "org.chromium.chrome.browser.sharing.proto";

package chrome_browser_sharing;

// Required in Chrome.
option optimize_for = LITE_RUNTIME;

// Message to pass phone number in click to call feature.
message ClickToCallMessage {
  // required
  string phone_number = 1;
}
